feat/CXSPA-7587:add help icon on shell bar link to help portal(#19019)

clordc opened the pull request:
Add an icon button "sys-help" on the shellbar which helps customers easy access to ASM help portal.
When customer clicks on this icon button, the browser will open up a new tab with the URL:
https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/d1ca9a1517f847a5b57b4e08400807c1/2367ff04ccb649ac82a3795ecf72a804.html?locale=en-US.

Commit:
Add help icon on shell bar link to help portal

i53577 commented on feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html:
```diff
@@ -98,3 +112,10 @@
 </ng-container>
+
+<style>
+  .fa-circle-question {
```
define style in *.scss file

clordc replied:
Styles are defined in the *.scss file

i53577 commented on feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html:
```diff
@@ -13,6 +13,20 @@
+        role="button"
+      >
+        <i class="fa-regular fa-circle-question"></i>
+        <span >{{ 'asm.helpPortalUrl' | cxTranslate }}</span></a
```
use  `npm run prettier:fix` or  `yarn prettier:fix` to format your code

clordc replied:
use npm run prettier:fix to format the code

i53577 commented on feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html:
```diff
@@ -13,6 +13,20 @@
     </div>
   </div>
   <div class="asm-bar-actions">
+    <div [ngStyle]="{'margin-right': '10px'}"
```
define style in *.scss file

clordc replied:
Styles are defined in the *.scss file

Commit:
define style in *.scss file

Commit:
change margin-right to padding-inline-end

Commit:
fix format problems

i53577 commented on feature-libs/asm/styles/components/_asm-main-ui.component.scss:
```diff
@@ -196,3 +196,10 @@
 }
+
+.cx-asm-help-portal {
+  padding-inline-end: 10px;
```
please use `rem` instead of `px`

clordc replied:
change px to rem

niehuayang commented on feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html:
```diff
@@ -13,6 +13,18 @@
+        rel="noopener noreferrer"
+        role="button"
+      >
+        <i class="fa-regular fa-circle-question"></i>
```
Where have you defined the class ‘fa-regular’?

clordc replied:
define the icon named help, which is sourced from fontawesome's far fa-circle-question.

niehuayang commented on feature-libs/asm/styles/components/_asm-main-ui.component.scss:
```diff
@@ -196,3 +196,10 @@
+
+.cx-asm-help-portal {
+  padding-inline-end: 10px;
+  .fa-circle-question {
```
the name of 'fa-circle-question' is strange, what's the fa?

clordc replied:
define the icon named help, which is sourced from fontawesome's far fa-circle-question

Commit:
change px to rem;
define the icon named help, which is sourced from fontawesome's far fa-circle-question;

Commit:
fix format problems

Commit:
Modified the spacing between icons in the interface

clordc committed:
Merge branch 'develop' into feature/CXSPA-7587

clordc committed:
Merge branch 'develop' into feature/CXSPA-7587

clordc committed:
Merge branch 'develop' into feature/CXSPA-7587

clordc committed:
Merge branch 'develop' into feature/CXSPA-7587
